src/Symfony/Component/Notifier/Bridge/Telnyx/README.md

```diff
@@ -0,0 +1,12 @@
+Telnyx Notifier
+===============
+
+Provides Telnyx integration for Symfony Notifier.
```

Comment:
Can you add a note about the DSN format (like done by most bridges)?

Reply:
Added